### PR TITLE
Sdkversionchange

### PR DESCRIPTION
**Pull request by rabollin**
Changed DAPR & CLI version to point to latest released version, also modified Go & Java SDK version to latest released version.

_Please explain the changes you've made_

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: endgame

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] The quickstart code compiles correctly
* [ ] You've tested new builds of the quickstart if you changed quickstart code
* [ ] You've updated the quickstart's README if necessary
* [ ] If you have changed the steps for a quickstart be sure that you have updated the automated validation accordingly. All of our quickstarts have annotations that allow them to be executed automatically as code. For more information see [mechanical-markdown](https://github.com/dapr/mechanical-markdown#readme). For user guide with examples see [Examples](https://github.com/dapr/mechanical-markdown/tree/main/examples#mechanical-markdown-by-example).

### CODE REVIEW

**Review by mukundansundar — APPROVED** [review]

**Comment by mukundansundar** [comment]
@dapr/maintainers-quickstarts Please review this PR.

**Comment by paulyuk** [comment]
taking it

File: pub_sub/go/sdk/checkout/go.sum
Comment (paulyuk) [inline pub_sub/go/sdk/checkout/go.sum:1]: I'm just double checking with @yaron2 that line 34 dependency on rc1 for protobuf makes sense.
`google.golang.org/protobuf v1.26.0-rc.1/go.mod h1:jlhhOSvTdKEhbULTjvd4ARK9grFBp09yW+WbY/TyQbw=`

**Review by paulyuk — APPROVED** [review]
LGTM, thanks for this @rabollin .  

Note we need one more PR still for `js-sdk` updates.

**Comment by paulyuk** [comment]
I'm going to take this now to unblock train.  If @yaron2 wants us to fix protobuf dependencies I'll do that in another PR.

**Pull request merged**
